src/server/server.go

```diff
@@ -59,10 +65,64 @@ func PprofMem(w http.ResponseWriter, r *http.Request) {
         runtime.GC()
 	w.Header().Add("Content-Type", "application/octet-stream")
         if err := pprof.WriteHeapProfile(w); err != nil {
-            log.Fatal("could not write memory profile: ", err)
+	    	log.Fatal("could not write memory profile: ", err)
         }
 }
 
+// Starts CPU profiling
+func PprofCpuStart(w http.ResponseWriter, r *http.Request) {
+	// user error: double start (previous profiling not stopped yet)
+	if cpuTemp != nil {
+		log.Printf("Already started cpu profiling\n")
```

Comment:
why not write this back to w along with the status code?  Probably more helpful to the user there than buried in the log output.